pkg/model/image_target.go

```diff
@@ -267,7 +267,8 @@ func (FastBuild) buildDetails()  {}
 func (fb FastBuild) Empty() bool { return reflect.DeepEqual(fb, FastBuild{}) }
 
 type CustomBuild struct {
-	Command string
+	TiltfilePath string
```

Comment:
can we call this workdir or something? To me, "TiltfilePath" means "/path/to/dir/Tilfile", but this is actually "/path/to/dir". And workdir matches the name in local_resource